toutoumomoma.go

```diff
@@ -328,31 +332,38 @@ func streamEntropy(r io.Reader) (float64, error) {
 		}
 		if err != nil {
 			if err != io.EOF {
-				return 0, err
+				return 0, 0, err
 			}
 			break
 		}
 	}
 
-	return entropy(&n, N), nil
+	h, sigma = entropy(&n, N)
```

Comment:
Nitpicking: "sigma" is usually used to denote the square root of variance. Can be misleading for the reader.

Reply:
Thanks.